test/integration/select_test.lua

```diff
@@ -96,6 +97,23 @@ local function insert_customers(g, customers)
     return inserted_objects
 end
 
+local function insert_goods(g, skus)
```

Comment:
This function looks like a copy-paste of `insert_customers`.

What about `insert_objects(g, space_name, objects)`?

Reply:
+

Reply:
fixed